commit 061fcfb46bf354a818f10db1bb0e2f02042dfe8a (refs/changes/14/514/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2016-09-22T02:20:21+00:00 (3 years, 1 month ago)
    
    MORAY-359 Moray doesn't handle queries that contain escaped backslashes properly
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/package.json b/package.json
index 5dd5033..bdd356d 100644
--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
         "deep-equal": "0.0.0",
         "fast": "0.3.6",
         "ip6addr": "0.1.1",
-        "ldap-filter": "0.2.0",
+        "ldap-filter": "0.2.4",
         "libuuid": "0.1.3",
         "lru-cache": "2.5.0",
         "node-manatee": "git+https://github.com/joyent/node-manatee.git#152d3e1",
